From a3d39ac8c2c8b0d1c7805c3a3515d0ea370b3a69 Mon Sep 17 00:00:00 2001
From: Jaga <jagadheesan_duraisamy@comcast.com>
Date: Wed, 19 Feb 2020 09:05:29 +0000
Subject: [PATCH] gupnp-binding-tool use python3
Source: COMCAST
License: LGPLv2+

Reason for change: Use python3 since it is default for dunfell

Upstream-Status: Pending
Signed-off-by: Jaga <jagadheesan_duraisamy@comcast.com>
---
 tools/gupnp-binding-tool | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/gupnp-binding-tool b/tools/gupnp-binding-tool
index 463e5ec..8abee73 100755
--- a/tools/gupnp-binding-tool
+++ b/tools/gupnp-binding-tool
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/python3
 
 # Copyright (C) 2008 OpenedHand Ltd
 # Copyright (C) 2008 Intel Corporation
